src/lib/trust_system/peer_trust.ml

```diff
@@ -171,7 +171,7 @@ module Make0 (Inputs : Input_intf) = struct
     let action_fmt, action_metadata = Action.to_log action in
     let log_trust_change () =
       let verb =
-        if simple_new.trust >. simple_old.trust then "Increasing"
+        if Float.( > ) simple_new.trust simple_old.trust then "Increasing"
```

Comment:
Are `>.` and friends gone from core now?

Reply:
There's a `Float.(=.)`, for sure (approximate equality), is there not a `Float.(>.)`?

Reply:
Yes, there is (this is v0.14, but should be the same in earlier versions): https://ocaml.janestreet.com/ocaml-core/latest/doc/core_kernel/Core_kernel/Float/index.html